modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -1423,6 +1426,124 @@ neo4j_home$ --nodes persons.csv --nodes games.csv --id-type string
+----
+
+Nodes are identified by their unique property value for the key/label combination that the header specifies.
+
+.Using actions in CSV files to update relationships
+[source, cypher, role="nocopy"]
+----
+:ACTION,:START_ID,:END_ID,:TYPE,role
+CREATE,person1,movie1,ACTED_IN,"Neo"
+UPDATE,person2,movie1,ACTED_IN,"Morpheus"
+DELETE,person3,movie1,ACTED_IN
+----
+
+Relationships are identified non-uniquely by their start and end node IDs, and their type.
+
+To further narrow down selection you can tag a property column as an identifier to help out in selecting relationships uniquely (or at least more uniquely).
+
+.Using actions in CSV files to update relationships with identifier properties
+[source, cypher, role="nocopy"]
+----
+:ACTION,:START_ID,:TYPE,:END_ID,p1{identifier:true},name,p4
+U,person1,KNOWS,person2,abc,"Keanu Reeves","Hello Morpheus"
+U,person2,KNOWS,person1,def,"Laurence Fishburne","Hello Neo"
+----
+
+The data in the `p1` column for these relationships acts "more uniquely" selecting relationships, if multiple of `1,KNOWS,2` exist.
+Identifier properties will match the selected relationships and as such not be set on the relationships (they already have it).
+
+=== Update existing labels
+
+You can add or remove one or more labels from an existing node by prepending the clause `LABEL` in the header with a `+` (default) or `-`:
+
+* `:+LABEL` - Add one or more labels to an existing node.
+* `:-LABEL` - Remove one or more labels (if they exist) from an existing node.
+
+
+For example, a file could have the following format:
+
+[source, csv]
+----
+uid:ID(label:Person),:+LABEL,:-LABEL,name,age
+person1,Actor,Producer,"Keanu Reeves",55
+person2,Actor;Director,"Laurence Fishburne",60
+----
+
+In this case, all labels in the second column are added and all the labels in the third column are removed (if they exist).
+
+=== Remove existing properties
+
+You can remove properties from existing nodes or relationships by a `:-PROPERTY` column in the header.
+In the contents of this field you can add zero or more property names to remove from the entity.
+All nodes are identified by their `:ID` and all relationships by their `:START_ID`, `:END_ID`, and `:TYPE`.
+For example:
+
+.Remove nodes' properties
+[source, cypher, role="nocopy"]
+----
+:ACTION,uid:ID(label:Person),:-PROPERTY
+U,person1,55;New York
+----
+
+Properties `55` and `New York` are removed from the node with the `uid:ID` `person1`.
+
+.Remove relationships' properties
+[source, cypher, role="nocopy"]
+----
+:ACTION,:START_ID,:END_ID,:TYPE,:-PROPERTY
+U,person1,movie1,ACTED_IN,Neo;"The chosen one"
+----
+
+Properties `Neo` and `"The chosen one"` are removed from the relationship with the `:START_ID` `person1`, `:END_ID` `movie1`, and `:TYPE` `ACTED_IN`.
+
+.Using actions in CSV files to update labels and properties
+[source, cypher, role="nocopy"]
+----
+:ACTION,uid:ID(label:Person),:LABEL,:-LABEL,:-PROPERTY,name,height:int
+U,person1,Actor,Producer,46;New York,"Henry",185
```

Comment:
Same problem with `:-PROPERTY` here, i.e. `45;New York` should instead be `age;hometown`